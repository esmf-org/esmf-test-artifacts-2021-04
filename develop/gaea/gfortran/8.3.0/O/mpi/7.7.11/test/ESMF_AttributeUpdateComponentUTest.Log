20210406 050302.654 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.654 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.654 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.654 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.654 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.654 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.654 INFO             PET0 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.654 INFO             PET0 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.654 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.654 INFO             PET0 ESMF_COMM                   : mpi
20210406 050302.654 INFO             PET0 ESMF_MOAB                   : enabled
20210406 050302.654 INFO             PET0 ESMF_LAPACK                 : enabled
20210406 050302.654 INFO             PET0 ESMF_NETCDF                 : enabled
20210406 050302.654 INFO             PET0 ESMF_PNETCDF                : disabled
20210406 050302.654 INFO             PET0 ESMF_PIO                    : enabled
20210406 050302.654 INFO             PET0 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET0  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET0  PASS  PET            0 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET0  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET0 Finalizing ESMF
20210406 050302.647 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.647 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.647 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.647 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.647 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.647 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.647 INFO             PET1 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.647 INFO             PET1 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.647 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.647 INFO             PET1 ESMF_COMM                   : mpi
20210406 050302.647 INFO             PET1 ESMF_MOAB                   : enabled
20210406 050302.647 INFO             PET1 ESMF_LAPACK                 : enabled
20210406 050302.647 INFO             PET1 ESMF_NETCDF                 : enabled
20210406 050302.647 INFO             PET1 ESMF_PNETCDF                : disabled
20210406 050302.647 INFO             PET1 ESMF_PIO                    : enabled
20210406 050302.647 INFO             PET1 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET1  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET1 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET1 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET1  PASS  PET            1 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET1  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET1 Finalizing ESMF
20210406 050302.630 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.631 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.631 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.631 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.631 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.631 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.631 INFO             PET2 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.631 INFO             PET2 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.631 INFO             PET2 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.631 INFO             PET2 ESMF_COMM                   : mpi
20210406 050302.632 INFO             PET2 ESMF_MOAB                   : enabled
20210406 050302.632 INFO             PET2 ESMF_LAPACK                 : enabled
20210406 050302.632 INFO             PET2 ESMF_NETCDF                 : enabled
20210406 050302.632 INFO             PET2 ESMF_PNETCDF                : disabled
20210406 050302.632 INFO             PET2 ESMF_PIO                    : enabled
20210406 050302.632 INFO             PET2 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET2  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET2  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET2 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET2 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET2  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET2  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET2  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET2  PASS  PET            2 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET2  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET2 Finalizing ESMF
20210406 050302.637 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.637 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.637 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.637 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.637 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.637 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.637 INFO             PET3 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.637 INFO             PET3 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.637 INFO             PET3 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.637 INFO             PET3 ESMF_COMM                   : mpi
20210406 050302.638 INFO             PET3 ESMF_MOAB                   : enabled
20210406 050302.638 INFO             PET3 ESMF_LAPACK                 : enabled
20210406 050302.638 INFO             PET3 ESMF_NETCDF                 : enabled
20210406 050302.638 INFO             PET3 ESMF_PNETCDF                : disabled
20210406 050302.638 INFO             PET3 ESMF_PIO                    : enabled
20210406 050302.638 INFO             PET3 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET3  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET3  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET3 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET3 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET3  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET3  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET3  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET3  PASS  PET            3 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET3  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET3 Finalizing ESMF
20210406 050302.638 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.638 INFO             PET4 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.638 INFO             PET4 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.638 INFO             PET4 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.638 INFO             PET4 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.638 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.638 INFO             PET4 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.638 INFO             PET4 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.638 INFO             PET4 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.638 INFO             PET4 ESMF_COMM                   : mpi
20210406 050302.639 INFO             PET4 ESMF_MOAB                   : enabled
20210406 050302.639 INFO             PET4 ESMF_LAPACK                 : enabled
20210406 050302.639 INFO             PET4 ESMF_NETCDF                 : enabled
20210406 050302.639 INFO             PET4 ESMF_PNETCDF                : disabled
20210406 050302.639 INFO             PET4 ESMF_PIO                    : enabled
20210406 050302.639 INFO             PET4 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET4  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET4  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET4 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET4 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET4  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET4  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET4  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET4  PASS  PET            4 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET4  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET4 Finalizing ESMF
20210406 050302.637 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.637 INFO             PET5 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.637 INFO             PET5 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.637 INFO             PET5 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.637 INFO             PET5 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.637 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.637 INFO             PET5 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.637 INFO             PET5 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.637 INFO             PET5 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.637 INFO             PET5 ESMF_COMM                   : mpi
20210406 050302.638 INFO             PET5 ESMF_MOAB                   : enabled
20210406 050302.638 INFO             PET5 ESMF_LAPACK                 : enabled
20210406 050302.638 INFO             PET5 ESMF_NETCDF                 : enabled
20210406 050302.638 INFO             PET5 ESMF_PNETCDF                : disabled
20210406 050302.638 INFO             PET5 ESMF_PIO                    : enabled
20210406 050302.638 INFO             PET5 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET5  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET5  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET5 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET5 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET5  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET5  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET5  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET5  PASS  PET            5 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET5  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET5 Finalizing ESMF
20210406 050302.632 INFO             PET6 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.632 INFO             PET6 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.632 INFO             PET6 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.632 INFO             PET6 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.632 INFO             PET6 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.632 INFO             PET6 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.632 INFO             PET6 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.632 INFO             PET6 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.632 INFO             PET6 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.632 INFO             PET6 ESMF_COMM                   : mpi
20210406 050302.632 INFO             PET6 ESMF_MOAB                   : enabled
20210406 050302.632 INFO             PET6 ESMF_LAPACK                 : enabled
20210406 050302.632 INFO             PET6 ESMF_NETCDF                 : enabled
20210406 050302.632 INFO             PET6 ESMF_PNETCDF                : disabled
20210406 050302.632 INFO             PET6 ESMF_PIO                    : enabled
20210406 050302.632 INFO             PET6 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET6  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET6  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET6 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET6 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET6  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET6  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET6  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET6  PASS  PET            6 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET6  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET6 Finalizing ESMF
20210406 050302.640 INFO             PET7 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.640 INFO             PET7 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210406 050302.640 INFO             PET7 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210406 050302.640 INFO             PET7 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210406 050302.640 INFO             PET7 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210406 050302.640 INFO             PET7 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210406 050302.640 INFO             PET7 Running with ESMF Version   : 8.2.0 beta snapshot
20210406 050302.640 INFO             PET7 ESMF library build date/time: "Apr  6 2021" "04:20:48"
20210406 050302.640 INFO             PET7 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O
20210406 050302.640 INFO             PET7 ESMF_COMM                   : mpi
20210406 050302.641 INFO             PET7 ESMF_MOAB                   : enabled
20210406 050302.641 INFO             PET7 ESMF_LAPACK                 : enabled
20210406 050302.641 INFO             PET7 ESMF_NETCDF                 : enabled
20210406 050302.641 INFO             PET7 ESMF_PNETCDF                : disabled
20210406 050302.641 INFO             PET7 ESMF_PIO                    : enabled
20210406 050302.641 INFO             PET7 ESMF_YAMLCPP                : enabled
20210406 050302.654 INFO             PET7  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
20210406 050302.654 INFO             PET7  NUMBER_OF_PROCESSORS           8
20210406 050302.688 DEBUG            PET7 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.689 DEBUG            PET7 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 1073741824 bytes
20210406 050302.736 INFO             PET7  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
20210406 050302.736 INFO             PET7  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
20210406 050302.736 INFO             PET7  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
20210406 050302.738 INFO             PET7  PASS  PET            7 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
20210406 050302.739 INFO             PET7  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
20210406 050302.739 INFO             PET7 Finalizing ESMF
