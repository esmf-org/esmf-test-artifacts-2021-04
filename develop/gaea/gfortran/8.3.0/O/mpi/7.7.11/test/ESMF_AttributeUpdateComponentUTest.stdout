build time -- 2021-04-27 16:21:44
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 448
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  NUMBER_OF_PROCESSORS           8
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 532
  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 539
  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 546
  PASS  PET            0 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            1 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            2 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            3 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            4 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            5 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            6 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  PASS  PET            7 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 553
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 578
  PET           1  Test Elapsed Time    72.372000000000000       msec.
  PET           2  Test Elapsed Time    72.583999999999975       msec.
  PET           3  Test Elapsed Time    72.597999999999999       msec.
  PET           4  Test Elapsed Time    63.293999999999990       msec.
  PET           5  Test Elapsed Time    63.387999999999998       msec.
  PET           6  Test Elapsed Time    62.847999999999985       msec.
  PET           7  Test Elapsed Time    63.533999999999978       msec.
  PET           0  Test Elapsed Time    55.874999999999993       msec.
